util/run-test.sh

```diff
@@ -34,22 +35,53 @@ show_coverage_summary () {
 }
 
 # TODO: refactor and move this to `Scripts.hs` when proven to work
```

Comment:
@yorkie You're still in the process of adjusting the test framework for DPS, right?